Sources/StreamVideoSwiftUI/CallView/LayoutComponents/DominantSpeakerLayoutComponent.swift

```diff
@@ -0,0 +1,84 @@
+//
+// Copyright © 2023 Stream.io Inc. All rights reserved.
+//
+
+import StreamVideo
+import SwiftUI
+
+/// `DominantSpeakerLayoutComponent` represents a view for the dominant speaker's thumbnail.
+public struct DominantSpeakerLayoutComponent<Factory: ViewFactory>: View {
```

Comment:
Similar comment about the name: dominant speaker is one way to be in the spotlight, but there are other ways as well, e.g. pinned. I think spotlight fits better here.